home_work_4/task2.py

```diff
@@ -0,0 +1,17 @@
+#   Задайте натуральное число N. Напишите программу, которая
+#   составит список простых множителей числа N.
+
+def mnojiteli(n):
+   i = 2
+   mnojiteli = []
+   while i * i <= n:
+       while n % i == 0:
+           mnojiteli.append(i)
+           n = n / i
```

Comment:
```suggestion
           n = n // i
```
Иначе последнее число будет в типе `float`.